integration/spark-common/src/main/scala/org/apache/spark/util/PartitionUtils.scala

```diff
@@ -223,4 +224,41 @@ object PartitionUtils {
       }
     }
   }
+
+  /**
+   * Used to extract Partitioner Fields for aggregation datamaps.
+   *
+   */
+  def getPartitionerFields(partitionColumn: Seq[String],
```

Comment:
Please add detail level comment for this method, 
Condition for generating partition filed